app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
```

Comment:
Code Style: As per the checklist, it's recommended to use descriptive and correct variable names. Here, `file_in` and `file_out` are not very descriptive. Consider renaming them to something more meaningful, like `source_file` and `destination_file`.